scripts/apps/authoring/attachments/actions.ts

```diff
@@ -0,0 +1,89 @@
+import {gettext} from 'core/utils';
+
+export const INIT_ATTACHMENTS = 'INIT_ATTACHMENTS';
+export function initAttachments(item) {
+    return (dispatch, geState, {deployConfig, attachments}) => {
```

Comment:
Named arguments would help. Where is `{deployConfig, attachments}` coming from? I thought only `dispatch, geState` was available in action creators.

Reply:
that's from [thunk](https://github.com/reduxjs/redux-thunk#injecting-a-custom-argument)

Reply:
I think calling it `thunkCustomArgument` and optionally destructing from it in the function body would be more clear.

Reply:
this whole thing with `return (dispatch, getState)` is from thunk, and it's a pattern we use in superdesk-planning as well, I would rather stick to it